users.json

```diff
@@ -0,0 +1,533 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1 - demo",
+        "title": "SCIM",
+        "description": "The SCIM API lets you manage users in your organization. You can then automate the provisioning of product licenses for these users, and they can use your company's Single Sign-On solution through an Identity Provider. The SCIM API lets you manage users in your organization. You can then automate the provisioning of product licenses for these users, and they can use your company's Single Sign-On solution through an Identity Provider."
+    },
+    "host": "pldgicrgsdfkmt6iq-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/Users/{userKey}": {
+            "parameters": [
+                {
```

Comment:
this should be the same definition used by the notification API - actually I believe we can refactor common definitions in shared definitions (e.g. a user ID)